deepchem/molnet/load_function/pdbbind_datasets.py

```diff
@@ -1,458 +1,124 @@
-      'random': deepchem.splits.RandomSplitter(),
-  }
-  splitter = splitters[split]
-  train, valid, test = splitter.train_valid_test_split(dataset, seed=split_seed)
+    ligand_files = [
+        os.path.join(data_folder, pdb, "%s_ligand.sdf" % pdb) for pdb in pdbs
+    ]
 
-  all_dataset = (train, valid, test)
-  print("\nSaving dataset to \"%s\" ..." % save_folder)
-  deepchem.utils.data_utils.save_dataset_to_disk(save_folder, train, valid,
-                                                 test, transformers)
-  return pdbbind_tasks, all_dataset, transformers
+    # Extract labels
+    with open(index_labels_file, "r") as g:
+      labels = np.array([
+          # Lines have format
+          # PDB code, resolution, release year, -logKd/Ki, Kd/Ki, reference, ligand name
+          # The base-10 logarithm, -log kd/pk
+          float(line.split()[3]) for line in g.readlines() if line[0] != "#"
+      ])
 
+    return (protein_files, ligand_files, labels, pdbs)
 
-def load_pdbbind_from_dir(data_folder,
-                          index_files,
-                          featurizer="grid",
-                          split="random",
-                          ex_ids=[],
-                          save_dir=None):
-  """Load and featurize raw PDBBind dataset from a local directory with the option to avoid certain IDs.
 
-    Parameters
-    ----------
-    data_dir: String,
-      Specifies the data directory to store the featurized dataset.
-    index_files: List
-      List of data and labels index file paths relative to the path in data_dir
-    split: Str
-      Either "random" or "index"
-    feat: Str
-      Either "grid" or "atomic" for grid and atomic featurizations.
-    subset: Str
-      Only "core" or "refined" for now.
-    ex_ids: List
-      List of PDB IDs to avoid loading if present
-    save_dir: String
-      Path to store featurized datasets
-    """
-  pdbbind_tasks = ["-logKd/Ki"]
+def load_pdbbind(
+    featurizer: dc.feat.ComplexFeaturizer,
+    splitter: Union[dc.splits.Splitter, str, None] = 'random',
+    transformers: List[Union[TransformerGenerator, str]] = ['normalization'],
+    reload: bool = True,
+    data_dir: Optional[str] = None,
+    save_dir: Optional[str] = None,
+    **kwargs
+) -> Tuple[List[str], Tuple[Dataset, ...], List[dc.trans.Transformer]]:
+  """Load PDBBind dataset.
 
-  index_file = os.path.join(data_folder, index_files[0])
-  labels_file = os.path.join(data_folder, index_files[1])
+  The PDBBind dataset includes experimental binding affinity data
+  and structures for 4852 protein-ligand complexes from the "refined set"
+  in PDBBind v2019. The refined set removes data with obvious problems
+  in 3D structure, binding data, or other aspects and should therefore
+  be a better starting point for docking/scoring studies. Details on
+  the criteria used to construct the refined set can be found in [4]_.
 
-  # Extract locations of data
-  pdbs = []
+  Random splitting is recommended for this dataset.
```

Comment:
This is out of scope for the current PR, but we might want to add a splitter based on sequence dissimilarity or potentially even an evolutionary splitter to measure generalizability across proteins in a future PR